contracts/Insurance.sol

```diff
@@ -126,7 +126,10 @@ contract Insurance is IInsurance, Ownable, SafetyWithdraw {
      * @param market The Tracer market whose insurance pool will be drained
      * @param amount The desired amount to take from the insurance pool
      */
-    function drainPool(address market, uint256 amount) external override onlyAccount() {
+    function drainPool(
```

Comment:
I'm assuming `onlyAccount` should be changed to something to do with liquidations? `onlyLiquidation`?

Reply:
Yep good catch @raymogg